templates/terraform/modules/s3_hosting/main.tf

```diff
@@ -128,7 +128,8 @@ resource "aws_cloudfront_distribution" "client_assets_distribution" {
 
   # Reference the cert validations only so it becomes a dependency
   tags = {
-    "certs-validated" = join(",", var.certificate_validations)
+    # join with realy used char - easier delimiter to indentify;
```

Comment:
A couple typos in here. Also how about just remove this comment and use a dash. The comment and the @ make it look like this has some special significance, but really people should pay attention only to the above comment which indicates that the value doesn't matter.